### ISSUE TITLE
Then value is not defined in #await block

### ISSUE THREAD

**Issue by MaxMilton** [sveltejs/eslint-plugin-svelte3#27]
In this code:

```html
<script>
  const loading = Promise.resolve(true);
</script>

{#await loading then show}
  {#if show}
    <div id="loading"></div>
  {/if}
{/await}

```

The `show` var in `#if show` is triggering the `no-undef` rule.

**Comment by Conduitry** [comment]
Yep, we're not looking at the appropriate field for `AwaitBlock` nodes in the AST to grab the variable we need to declare for this scope. One wrinkle though is that this field is currently always a string (because there is no await block destructuring yet). I can _guess_ at what this field might look like in the future when that's added (I would guess that the AST would look like that in an each block), but there might be another change I need to remember to make here when that happens.

**Comment by Conduitry** [comment]
Should be fixed in 2.2.1.
